resources/test262-harness.js

```diff
@@ -0,0 +1,39 @@
+// Source: https://github.com/bakkot/test262-web-runner
+function installAPI(global) {
+	global.$262 = {
+		createRealm: function() {
+			var iframe = global.document.createElement('iframe');
+            iframe.style.cssText = "display: none";
+			iframe.src = ""; // iframeSrc;
+			global.document.body.appendChild(iframe);
+			return installAPI(iframe.contentWindow);
+		},
+		evalScript: function(src) {
+			var script = global.document.createElement('script');
+			script.text = src;
+			global.document.body.appendChild(script);
+		},
+		detachArrayBuffer: function(buffer) {
+			if (typeof postMessage !== 'function') {
+				throw new Error('No method available to detach an ArrayBuffer');
+			} else {
+                postMessage(null, '*', [buffer]);
```

Comment:
Strange indentation 0_o

Reply:
It seems there's a mix up of tabs and spaces in that file. I will review all the files and make sure there are only spaces.